ci/Jenkinsfiles/build.groovy

```diff
@@ -643,7 +642,8 @@ pipeline {
             git config credential.helper store
 
             # Git tag
-            jx step tag -v ${VERSION}
+            git tag -a v${VERSION} -m "Release ${VERSION}"
+            git push origin v${VERSION}
```

Comment:
The original jx step tag adds a --force (-f) to the `git tag` call. I guess it's unecessary ? 
And calling the git command manually spares you commiting again uselessly ?

Reply:
> The original jx step tag adds a --force (-f) to the git tag call. I guess it's unnecessary ?

Trying to replace an exiting tag should never happen and I think we prefer an explicit error if ever it were the case.

> And calling the git command manually spares you committing again uselessly ?

Yes it prevents from having an unnecessary extra empty commit, see for instance https://github.com/nuxeo/nuxeo/commit/90b1aaa37df5afb8610a8e89ce580817835aecad and
![image](https://user-images.githubusercontent.com/1265714/84015532-495b9900-a97c-11ea-816a-e2319c2c731b.png)

And its one less use of `jx` :smile:.

Reply:
Thanks for explenations !